playbooks/kvm-deploy_caasp.yml

```diff
@@ -43,3 +43,12 @@
       vars:
         skuba_cluster_basedir: "{{ socok8s_workspace }}"
         image_username: "{{ skuba_ci_terraform_libvirt_image_username }}"
+
+- hosts: caasp-workers
+  vars_files:
+    - "{{ playbook_dir }}/../vars/common-vars.yml"
+  tasks:
+    - import_role:
```

Comment:
just make it a role, instead of tasks -import_role ?

Reply:
done!